src/components/AddressSearch/index.tsx

```diff
@@ -412,18 +452,22 @@ function AddressSearch(
                         styles={{
                             textInputContainer: [styles.flexColumn],
                             listView: [StyleUtils.getGoogleListViewStyle(displayListViewBorder), styles.overflowAuto, styles.borderLeft, styles.borderRight, !isFocused && {height: 0}],
-                            row: [styles.pv4, styles.ph3, styles.overflowAuto],
+                            row: [styles.overflowAuto],
                             description: [styles.googleSearchText],
                             separator: [styles.googleSearchSeparator],
                         }}
                         numberOfLines={2}
                         isRowScrollable={false}
                         listHoverColor={theme.border}
                         listUnderlayColor={theme.buttonPressedBG}
-                        onLayout={(event: LayoutChangeEvent) => {
+                        onLayout={(event) => {
                             // We use the height of the element to determine if we should hide the border of the listView dropdown
                             // to prevent a lingering border when there are no address suggestions.
                             setDisplayListViewBorder(event.nativeEvent.layout.height > variables.googleEmptyListViewHeight);
+                            const {target} = event.nativeEvent as unknown as BaseSyntheticEvent;
```

Comment:
I think there's probably a better way to fix this type. I see `event.target` is valid, it's just missing the `getScrollResponser()` function. Maybe need to use generics like `LayoutChangeEvent<GooglePlacesAutoComplete>` or something

Reply:
I tried LayoutChangeEvent<GooglePlacesAutoComplete> but getting error Type 'LayoutChangeEvent' is not generic. The event type LayoutChangeEvent is defined as NativeSyntheticEvent<{layout: LayoutRectangle}>. So most probably the event.nativeEvent would need to be casted to BaseSyntheticEvent to get to the target.

Reply:
I tried LayoutChangeEvent but getting error Type 'LayoutChangeEvent' is not generic. The event type LayoutChangeEvent is defined as NativeSyntheticEvent<{layout: LayoutRectangle}>. So the event.nativeEvent would need to be casted to BaseSyntheticEvent to get to the target prop.

Reply:
@sourcecodedeveloper Sorry, I added this comment in last Review, but it didn't come up. 

Anyways, I think the concern here is valid, the type of `const {target}` (target) is resolving `any` in this place which would be bad representation in typescript.

Reply:
@abdulrahuman5196 I was able to leverage generics to get type for target prop and not use any.